CONTRIBUTING.md

```diff
@@ -1,45 +1,111 @@
 # Contributing
 
-If you use jrnl, you can totally make our day by just saying "thanks for the code." It's your chance to make a programmer happy today! If you have a moment, let us know what you use jrnl for and how; it'll help us to make it even better!
+We welcome contributions to jrnl, whether it's through reporting bugs, improving the documentation, testing releases, engaging in discussion on features and bugs, or writing code.
 
+## Table of Contents
+ * [Code of Conduct](#code-of-conduct)
+ * [Reporting Bugs](#reporting-bugs)
+ * [Editing Documentation](#editing-documentation)
+ * [Testing](#testing)
+ * [Submitting feature requests and ideas](#submitting-feature-requests-and-ideas)
+ * [Developing](#developing)
 
-# Table of Contents
- * [Docs and Typos](#docs-and-typos)
- * [Bugs](#bugs)
- * [Feature requests and ideas](#feature-requests-and-ideas)
- * [New programmers and programmers new to python](#new-programmers-and-programmers-new-to-python)
- * [Developing jrnl](#developing-jrnl)
+## Code of Conduct
 
+Before starting, please read the [Code of Conduct](CODE_OF_CONDUCT.md).
 
-## Docs and Typos
+## Reporting Bugs
 
-If you find a typo or a mistake in the docs, please fix it right away and send a pull request. The Right Way™ to fix the docs is to edit the `docs/*.md` files on the **master** branch. You can see the result if you run `make html` inside the project's root directory, which will open a browser that hot-reloads as you change the docs. This requires [mkdocs](https://www.mkdocs.org) to be installed. The `gh-pages` branch is automatically maintained and updates from `master`; you should never have to edit that.
+Please report bugs by [opening a new issue](https://github.com/jrnl-org/jrnl/issues/new/choose) and describing it as well as possible. Many bugs are specific to a particular operating system and Python version, so details are encouraged.
```

Comment:
The rest of this PR has a great, personal voice, but 'details are encouraged' here is very passive. Can we change to something more personal?